src/merkleDistributor/model/MerkleTree.ts

```diff
@@ -0,0 +1,126 @@
+// This script provides some useful methods for building MerkleTrees. It is taken verbatim from the uniswap implementation
```

Comment:
Random comment, can you explain the meaning of the "model" naming of this directory? I could see types or utils potentially being more clear?

Reply:
It's a synonym for types, yes, but I like it because it's a more generic word. `type` can be misleading in a JS program. Basically in the `model` dir you can find types, classes, enums, etc